drake/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 2.8.10)
 
+add_matlab_test(NAME jenkins_junk_test COMMAND jenkins_launch_matlab)
+
+message(FATAL_ERROR "exiting")
+
```

Comment:
This won't work.   you need to make it 'run" the test at cmake time.   I guess you could just put a return() in here, and the configure would finish and then testing would work.   You could try this on your mac first.